### DISCUSSION TITLE
使用剪贴板 - forecho's Blog

### DISCUSSION THREAD

**Discussion by giscus** [forecho/hugo-blog#21] [Announcements]
# 使用剪贴板 - forecho's Blog

引言
剪贴板是一个非常有效率的工具，自从 2015 年无意中从同事那里看到了这个工具，我就一直在使用它。用了再也离不开的工具，没它就很不放心。
这篇文章就简单聊聊我所知道的剪贴板工具，并简单做个测评。
测评
Ditto

我接触到的第一个剪贴板工具就是 Ditto，它是一个开源的剪贴板工具，它的功能非常强大，界面很简洁。
但是它只支持 Windows，Ditto 作为我的第一个剪贴板工具，几年下来我的操作习惯深受其影响。

https://blog.forecho.com/use-pasteboard.html

**Comment by baixiang-main** [comment]
[CopyQ](https://hluk.github.io/CopyQ/)也不错，看了下官网，支持 Linux, Windows and OS X 10.15+。使用观感上很接近Ditto，我是从Ditto换到了CopyQ目前。

**Reply by forecho** [reply]
感谢分享，第一次听说 CopyQ

**Comment by dongyubin** [comment]
之前整理的[剪贴板历史管理软件](https://www.wangdu.site/software/294.html)，现在mac在用Alfred自带的剪贴板，用着也不错

**Comment by carlfengo** [comment]
可以试试快贴，支持很多平台同步，一直在用，免费。
